backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardlist/stack/BpkStackCardList.kt

```diff
@@ -0,0 +1,59 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2025 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.cardlist.stack
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.cardlist.stack.internal.BpkStackCardListImpl
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.sectionheader.BpkSectionHeaderButton
+
+sealed class BpkStackCardAccessoryStyle {
+    data class Expand(
+        val collapsedCount: Int,
+        val expandedCount: Int,
+        val expandText: String,
+        val collapsedText: String,
+        val onExpansionChange: ((Boolean) -> Unit)? = null,
+    ) : BpkStackCardAccessoryStyle()
+    data class Button(val title: String, val icon: BpkIcon, val onClick: () -> Unit) : BpkStackCardAccessoryStyle()
+}
+
+@Composable
+fun BpkStackCardList(
+    title: String,
```

Comment:
@xo0ps looking at this [figma design](https://www.figma.com/design/LaYKZu5Utu80iwnB5xKATE/Homepage---MVP?node-id=1604-41314&p=f&t=q6S7likgR0eqr3pj-0)  
you may need to change the title to something else than string as you could display `bpkIcon` as a prefix later on and not just a plain title. it is worth checking with design about this as [Card List backpack figma](https://www.figma.com/design/irZ3YBx8vOm16ICkAr7mB3/Backpack-Components?node-id=33783-33859&p=f&t=vDcPUCB0AZNyuD8F-0) doesn't show that